auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -306,14 +298,10 @@ private Path pathFor(String name) {
                     .toLowerCase(Locale.ROOT)
                     .replaceAll("[^a-z0-9]", "_");
         }
-        String base32 = ByteIterator.ofBytes(new ByteStringBuilder().append(name).toArray())
-                .base32Encode(Base32Alphabet.STANDARD, false).drainToString();
-        if (this.secretKey != null) {
-            normalizedName = base32;
-        }
-
-        if (encoded) {
-            normalizedName = normalizedName + "-" + base32;
+        if (secretKey != null | encoded) {
```

Comment:
Sorry, I had a typo in my original comment for this one. We should use `||` here (instead of just `|`).

Reply:
Fixed